### PR TITLE
[Dominion] Releases per country

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

Currently release dropdown contains all countries releases. This is a fix to show specific country release option on dropdown

Fixes # (issue)

- [x] Bug fix (non-breaking change which fixes an issue)

### Before

Screenshot from 2019-03-20 13-45-10

### After

Screenshot from 2019-03-20 13-44-36

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: dominion/views.py [hunk 1]
```diff
@@ -20,3 +29,71 @@ def get_context_data(self, **kwargs):
         context['country'] = country
         context['geography'] = geo.as_dict_deep()
         return context
+
+class GeographyDetailView(BaseGeographyDetailView):
+    adjust_slugs = True
+    default_geo_version = None
+
+    def dispatch(self, *args, **kwargs):
+        request = args[0]
```
Comment (karimkawambwa) [inline dominion/views.py]: minor: `dispatch(self, request, *args, **kwargs)`

File: dominion/views.py [hunk 2]
```diff
@@ -20,3 +29,71 @@ def get_context_data(self, **kwargs):
...
+        page_context['head2head'] = 'h2h' in self.request.GET
+
+        return page_context
+
+    def get_geography(self, geo_id):
+        # stub this out to prevent the subclass for calling out to CR
+        pass
+
+    def get_template_names(self):
+        return ['profile/profile_detail_%s.html' % self.profile_name, 'profile/profile_detail.html']
```
Comment (karimkawambwa) [inline dominion/views.py:109]: not sure what this overrides, i don't think we have template names with the `profile_name`

**Review by karimkawambwa — CHANGES_REQUESTED** [review]
I think some imports are needed urllib, Http404, redirect ...

**Comment by karimkawambwa** [comment]
Other than that from what I could gather and understand, looks LGTM 🚀 after the imports.

File: dominion/views.py (hunk 1 above)
Comment (kilemensi) [reply dominion/views.py]: It's wazimap style but fixed @karimkawambwa

File: dominion/views.py (hunk 2 above)
Comment (kilemensi) [reply dominion/views.py:109]: Yeah we don't use it now @karimkawambwa. But my understanding is this is for when you have multiple profiles, say Census & Something else. You could then use `profile_detail_xxx` to setup templates for each profile.

**Pull request merged**